src/components/Layouts/Primary/TopBar/Items/AddMenu/Items/CreateMultipleContacts/CreateMultipleContacts.test.tsx

```diff
@@ -131,20 +138,25 @@ describe('CreateMultipleContacts', () => {
       userEvent.type(getAllByRole('combobox')[0], address);
       userEvent.type(getAllByRole('textbox', { name: 'Phone' })[0], phone);
       userEvent.type(getAllByRole('textbox', { name: 'Email' })[0], email);
+      userEvent.type(
+        getAllByRole('listbox', { name: 'Status' })[0],
+        statusTextName,
+      );
       await waitFor(() => expect(getByText('Save')).not.toBeDisabled());
       userEvent.click(getByText('Save'));
       await waitFor(() => expect(handleClose).toHaveBeenCalled());
 
-      const { operation } = mutationSpy.mock.calls[0][0];
+      const { operation } = mutationSpy.mock.calls[1][0];
```

Comment:
A bunch of these in this file could be updated to use `expect().toHaveGraphqlOperation` so that we never have to go through and update call indexes again.